### DISCUSSION TITLE
Get liquid cloud water on native hybrid levels

### DISCUSSION THREAD

**Discussion by csteele2** [NOAA-GSL/GSL-Regional-Model-Forum#38] [Q&A] [answered]
Greetings!

I am attempting to get liquid cloud water on native hybrid levels, using the grib2 archive on AWS. The variables in here appear to be slightly different than on nomads. In looking at an [index file]([hrrr.t00z.wrfnatf00.grib2.idx](https://noaa-hrrr-bdp-pds.s3.amazonaws.com/hrrr.20230330/conus/hrrr.t00z.wrfnatf00.grib2.idx)), I noticed there is a "CLMR" variable. On nomads (at least via the grib filter), I can get CLWMR. What all goes into the CLMR variable in the native levels on AWS? Is that total of everything (and thus would have to subtract out ice, snow, etc.), or all liquid (and would have to subtract out rain)?

Thanks in advance!

**Comment by EricJames-NOAA** [comment, marked as answer]
Hi Caleb!  I assume you're referring to the HRRR?  I think the cloud mixing ratio field gets labeled as CLMR or CLWMR depending on the GRIB2 encoding specifications in UPP, but they are the same field.  It's the cloud water mixing ratio, so does not include any frozen component, or any precipitation.  Hopefully that answers your question.

**Reply by csteele2** [reply]
Yes, it was the HRRR. And that does. Oddly enough, when I open the file with xarray/cfgrib, the short name was CLWMR. Thanks!
